gtk4/src/dialog.rs

```diff
@@ -36,9 +36,27 @@ pub trait DialogExtManual: 'static {
 
     #[doc(alias = "gtk_dialog_get_response_for_widget")]
     fn get_response_for_widget<P: IsA<Widget>>(&self, widget: &P) -> ResponseType;
+
+    // rustdoc-stripper-ignore-next
+    /// Shows the dialog and calls the callback when a response has been received.
+    ///
+    /// **Important**: this function isn't blocking.
+    ///
+    /// ```no_run
+    /// let dialog = gtk::MessageDialogBuilder::new()
+    ///    .buttons(gtk::ButtonsType::OkCancel)
+    ///    .text("What is your answer?")
+    ///    .build();
+    ///
+    /// dialog.run(|obj, answer| {
+    ///     obj.close();
+    ///     println!("Answer: {:?}", answer);
+    /// });
+    /// ```
+    fn run<F: Fn(&Self, ResponseType) + 'static>(&self, f: F);
```

Comment:
```suggestion
    fn run<F: FnOnce(&Self, ResponseType) + 'static>(&self, f: F);
```

This is useless otherwise